app/src/main/java/com/keylesspalace/tusky/db/TimelineDao.kt

```diff
@@ -74,12 +74,12 @@ AND
     abstract fun removeAllPlaceholdersBetween(account: Long, maxId: String, sinceId: String)
 
     @Query("""UPDATE TimelineStatusEntity SET favourited = :favourited
-WHERE timelineUserId = :accountId AND (serverId = :statusId OR reblogServerId - :statusId)""")
+WHERE timelineUserId = :accountId AND (serverId = :statusId OR reblogServerId = :statusId)""")
```

Comment:
what was the minus doing here :no_mouth:

Reply:
:man_facepalming: 
How on Earth did it work. How did it pass tests. Ughouhg.